src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,27 @@
+.stars {
+  display: flex;
+  width: 96px;
+  justify-content: space-between;
+  align-items: center;
+
+  &__star {
+    background-image: url('../images/star.svg');
+    background-position: 50%;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+
+    &.active {
+      background-image: url('../images/star-active.svg');
+    }
+  }
+
+  &--0 .stars__star:nth-child(-n + 0),
+  &--1 .stars__star:nth-child(-n + 1),
+  &--2 .stars__star:nth-child(-n + 2),
+  &--3 .stars__star:nth-child(-n + 3),
+  &--4 .stars__star:nth-child(-n + 4),
+  &--5 .stars__star:nth-child(-n + 5) {
+    background-image: url('../images/star-active.svg');
+  }
```

Comment:
The `background-image` property is repeated in your code. Consider using a loop to generate the classes for `&--0` to `&--5`. This will make your code more DRY (Don't Repeat Yourself). You can use the `@for` directive in SCSS to achieve this.